src/styles/normilize.scss

```diff
@@ -0,0 +1,15 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
```

Comment:
I don't see these tags being used in your code, do you really need to set a css rule for them?